README.md

```diff
@@ -26,16 +26,18 @@ The goal of the engine is to have as much customization as possible, while still
 
 ## Supported Compilers
 
-| Compiler (Latest) | Windows            | Linux     |
-|-------------------|--------------------|-----------|
-| GCC               | Supported (MinGW)* | Supported |
-| Clang             | Unsupported        | Supported |
-| MSVC              | Supported          | N/A       |
-| Clang-CL          | Supported          | N/A       |
+| Compiler (Latest) | Windows            | Linux     | macOS                      |
+|-------------------|--------------------|-----------|----------------------------|
+| GCC               | Supported (MinGW)\* | Supported | Unsupported                |
+| Clang             | Unsupported        | Supported | Supported (Homebrew)\*\*|
+| MSVC              | Supported          | N/A       | N/A                        |
+| Clang-CL          | Supported          | N/A       | N/A                        |
 
-(*) MinGW builds on Windows will work, but running the application outside your IDE will require you to copy some DLLs
+(\*) MinGW builds on Windows will work, but running the application outside your IDE will require you to copy some DLLs
 next to the executable. Check the GitHub Actions script to see which DLLs need copied from the MinGW bin directory.
 
+(\*\*) Homebrew LLVM/Clang is required because Apple Clang as of this time does not support certain C++ features that are used inside ChiraEngine
```

Comment:
```suggestion
(\*\*) Homebrew LLVM/Clang is required because Apple Clang as of this time does not support certain C++ features that are used inside the engine.
```